pkg/network/node/ovscontroller.go

```diff
@@ -34,7 +34,7 @@ const (
 	Vxlan0 = "vxlan0"
 
 	// rule versioning; increment each time flow rules change
-	ruleVersion = 11
+	ruleVersion = 12
```

Comment:
I don't think this is a problem on 4.9 -> 4.10 upgrades, because we'll do `ruleVersion` 11/12 -> 12 upgrades. But I am not sure what to do on the 4.8 PR which is coming up: if we bump the `ruleVersion` in the same way, doesn't that mean that the possibility for some customers to upgrade from `ruleVersion` 12 (latest 4.8.z with this fix and `ruleVersion` 12) -> 11 (4.9.z without this fix, and `ruleVersion` 11) exists?

Reply:
Yeah, we probably ought to include the OCP version in the `ruleVersion`...

> the possibility for some customers to upgrade from ruleVersion 12 (latest 4.8.z with this fix and ruleVersion 12) -> 11 (4.9.z without this fix, and ruleVersion 11) exists?

It should be fixed in 4.9.z before it is fixed in 4.8.z, so it should not be possible to upgrade from 4.8.z with the fix to 4.9.z without the fix.

At any rate, we check `ruleVersion` with `==`, not `>`, so if you did go from a `ruleVersion = 12` release to a `ruleVersion = 11` release, it will work.

Reply:
> (The problem would come when ruleVersions get out of sync on different branches, where ruleVersion = 12 on 4.8 means something different from ruleVersion = 12 on 4.9.)

Got it. Shouldn't be concerned here then since we've bumped the rule version correctly